assessments/CLI_TX_KEA/README.md

```diff
@@ -0,0 +1,34 @@
+This is an earthmover bundle created from the following Ed-Fi Data Import Tool mapping:
+* **Title**: Texas Kindergarten Entry Assessment (TX-KEA) Assessment Results - API 3.X
+* **Description**: This template maps CLI TX-KEA Assessment Results, providing a comprehensive evaluation of various learning domains crucial for academic success at kindergarten entry and throughout the kindergarten year.
+    
+    For more info on these data files, see https://public.cliengage.org/tools/assessment/tx-kea/
+
+* **API version**: 5.3
```

Comment:
@rlittle08 how do I decipher the right API Version to use, both on this line of code, and on line [#2](https://github.com/edanalytics/earthmover_edfi_bundles/pull/34/files#diff-94fa5636ac292196ced785a39a40b54de0c6af32d419979615cdad0fca38fa13R2)?